getsolus/packages
# nvtop: Initial inclusion

**Summary**

- Initial inclusion of nvtop
- Resolves https://github.com/getsolus/packages/issues/1335

**Test Plan**

- Open `nvtop` and run `glxgears`
- Make sure activity is shown on `nvtop` (Tested on AMD RX550)

**Checklist**
